package.json

```diff
@@ -86,11 +86,14 @@
     "mocha-junit-reporter": "^1.17.0",
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0",
-    "webpack": "^5.58.1"
+    "webpack": "^5.75.0"
   },
   "peerDependencies": {
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "webpack": "^5.58.1"
+    "webpack": "^5.75.0"
```

Comment:
@mkuklis, stripes-cli can't satisfy this because [it's pinned to ~5.68.0](https://github.com/folio-org/stripes-cli/blob/820959337b88aa38d820ef63f2422df774a006ed/package.json#L67) due to PR https://github.com/folio-org/stripes-cli/pull/284. I know you said this build was successful locally. Did you try running karma/BTOG tests in any repo with this change in place? 

How I would looooooove for this to be BTOG's death knell. If only it were so easy.

Reply:
@zburke I haven't tried that. Let me just put it back to what it was and we can merge it.